srcpkgs/nodejs/template

```diff
@@ -93,7 +96,7 @@ do_check() {
 }
 
 do_install() {
-	make LD="$CXX" LDFLAGS+=-ldl DESTDIR="$DESTDIR" install
+	make LD="$CXX" LDFLAGS+=-ldl DESTDIR="$DESTDIR" ${makejobs} install
```

Comment:
make install built some stuff on my machine and maxed out CPU as a result (I use -j2 on a system with 4 logical cores), which is why I added this.